### PR TITLE
PXB-2694 Add the KMIP description to documentation 

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified: source/advanced/encrypted_innodb_tablespace_backups.rst

### CODE REVIEW

File: storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst [hunk 1]
```diff
@@ -353,6 +355,22 @@ The backup is now prepared and can be restored with :option:`--copy-back` option
...
+
+Percona XtraBackup 8.0.27-19 adds support for the Key Management Interoperability Protocol (KMIP) which enables the communication between the key management system and encrypted database server.
+
+KMIP has no special requirements for backing up a database that contains encrypted InnoDB tablespaces. 
+
+Percona XtraBackup performs the following actions:
+
+1. Connects to the MySQL server
+2. Pulls the configuration
+3. Connect to the KMIP server
```
Comment (patrickbirch) [inline storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst]: Connects

File: storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst [hunk 2]
```diff
@@ -353,6 +355,22 @@ The backup is now prepared and can be restored with :option:`--copy-back` option
...
 used to take and prepare the backup.
 
+Using Key Management Interoperability Protocol
+==============================================
+
+This feature is *tech preview* quality.
+
+Percona XtraBackup 8.0.27-19 adds support for the Key Management Interoperability Protocol (KMIP) which enables the communication between the key management system and encrypted database server.
+
+KMIP has no special requirements for backing up a database that contains encrypted InnoDB tablespaces.
```
Comment (patrickbirch) [inline storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst]: Percona XtraBackup?

File: storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst (hunk 1 above)
Comment (alina-derkach-oaza) [reply storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst]: fixed

File: storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst (hunk 2 above)
Comment (alina-derkach-oaza) [reply storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst]: fixed

File: storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst [hunk 3]
```diff
@@ -5,9 +5,7 @@ Encrypted InnoDB tablespace backups
 ===================================
 
 InnoDB supports `data encryption for InnoDB tables
-<http://dev.mysql.com/doc/refman/8.0/en/innodb-tablespace-encryption.html>`_
-stored in file-per-table tablespaces. This feature provides an at-rest encryption
-for physical tablespace data files.
+<https://dev.mysql.com/doc/mysql-enterprise-backup/8.0/en/meb-encrypted-innodb.html>`__ stored in file-per-table tablespaces. This feature provides an at-rest encryption for physical tablespace data files.
```
Comment (rahulmalik87) [inline storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst]: seems like pointing wrong. may be https://dev.mysql.com/doc/refman/8.0/en/innodb-data-encryption.html

**Review by rahulmalik87 — CHANGES_REQUESTED** [review]

File: storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst (hunk 3 above)
Comment (alina-derkach-oaza) [reply storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst]: fixed

File: storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst [hunk 4]
```diff
@@ -353,6 +355,22 @@ The backup is now prepared and can be restored with :option:`--copy-back` option
...
+Percona XtraBackup 8.0.27-19 adds support for the Key Management Interoperability Protocol (KMIP) which enables the communication between the key management system and encrypted database server.
+
+Percona XtraBackup has no special requirements for backing up a database that contains encrypted InnoDB tablespaces. 
+
+Percona XtraBackup performs the following actions:
+
+1. Connects to the MySQL server
+2. Pulls the configuration
+3. Connects to the KMIP server
+4. Fetches the necessary keys from the KMIP server.
```
Comment (satya-bodapati) [inline storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst]: Are these keys stored in the backup directory? Good to have this info

**Review by satya-bodapati — CHANGES_REQUESTED** [review]

File: storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst (hunk 4 above)
Comment (alina-derkach-oaza) [reply storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst]: Updated

**Review by satya-bodapati — APPROVED** [review]
LGTM!

**Review by patrickbirch — APPROVED** [review]

**Pull request merged**